authentication/src/main/kotlin/com/uber/sdk2/auth/request/Prompt.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2024. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.uber.sdk2.auth.request
+
+/**
+ * Represents the prompt parameter for the OAuth 2.0 authorization request.
+ * https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest:~:text=select_account-,The,-Authorization%20Server%20SHOULD
+ */
+enum class Prompt(val value: String) {
```

Comment:
Why the `value`?

Reply:
Nevermind, I see it in `authContext.prompt?.let { put(UriConfig.PROMPT_PARAM, it.value) }`